osidb/api_views.py

```diff
@@ -345,6 +354,20 @@ def create(self, request, *args, **kwargs):
         return response
 
 
+@include_meta_attr_extend_schema_view
+@include_exclude_fields_extend_schema_view
+@extend_schema_view(
+    create=extend_schema(
+        request=FlawAcknowledgmentPostSerializer,
+    ),
+)
+class FlawAcknowledgmentView(ModelViewSet):
```

Comment:
This ViewSet does not have the destroy method overridden and thus the delete will be perform only on the OSIDB side and not on the BZ side. Similar handling as for Flaw Affects or Flaw References should be done.

Also I wonder whether it does make sense to create some common mixin which would already provide the overridden delete.

Reply:
Fixed in commit 3c2d2dd147380fc8924fac4b9d539f85f86f36d4 except for the new mixin. I'd prefer to do the mixin in a separate MR once we get an approval on this MR because of potential complexity, because it mostly depends on whether we agree on PR265, and because of the iterative approach we discussed two weeks ago.

Reply:
Add "common mixin" - it was not said that we should do it in this PR. I think it was only a suggestion for the future :).

Reply:
This is blocking other work (OSIDB-1004) and it seems the concerns have been addressed. If a common mixin is still needed, please open a separate Jira, or I'll try to implement it as a part of future similar work (OSIDB-1006 potentially).